v3/src/components/graph/adornments/movable-point/movable-point-adornment-handler.ts

```diff
@@ -0,0 +1,42 @@
+import { DIAdornmentHandler } from "../../../../data-interactive/handlers/adornment-handler"
+import { IGraphContentModel } from "../../models/graph-content-model"
+import { IAdornmentModel } from "../adornment-models"
+import { isMovablePointAdornment } from "./movable-point-adornment-model"
+import { AdornmentData, cellKeyToCategories } from "../utilities/adornment-handler-utils"
+import { kMovablePointType } from "./movable-point-adornment-types"
+
+export const movablePointAdornmentHandler: DIAdornmentHandler = {
+  get(adornment: IAdornmentModel, graphContent: IGraphContentModel) {
+    if (!isMovablePointAdornment(adornment)) {
+      return { success: false, values: { error: `Not a ${kMovablePointType} adornment` } }
+    }
+
+    const dataConfig = graphContent.dataConfiguration
+    const cellKeys = dataConfig?.getAllCellKeys()
+    const allAttributes = dataConfig?.dataset?.attributes
+    const xAttrId = dataConfig?.attributeID("x") || ""
+    const yAttrId = dataConfig?.attributeID("y") || ""
+    const xAttr = allAttributes?.find(attr => attr.id === xAttrId)
+    const yAttr = allAttributes?.find(attr => attr.id === yAttrId)
```

Comment:
I think this should work. Then you can remove line 16 above.
```suggestion
    const xAttr = dataConfig?.dataset?.getAttribute(xAttrId)
    const yAttr = dataConfig?.dataset?.getAttribute(yAttrId)
```